back/spec/acceptance/ideas_spec.rb

```diff
@@ -189,7 +189,7 @@
           basket = create(:basket)
           [@ideas[1], @ideas[2], @ideas[5]].each { _1.baskets << basket }
 
-          do_request(basket: basket.id)
+          do_request(basket_id: basket.id)
```

Comment:
Should we change it in line 69 too? https://github.com/CitizenLabDotCo/citizenlab/pull/4698/files#diff-d211418c2d7d65bb934abf64fc154b037205e64577a1b69e32d73d43fc731605R69